python-backend/app/api/variances/resources/variance_resource.py

```diff
@@ -43,24 +60,70 @@ def get(self):
             'total': pagination_details.total_results,
         }
 
+    @classmethod
```

Comment:
Should this be a staticmethod instead? Doesn't look like you're using the class instance at all.

Or even a regular internal method like the filter method